app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,37 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find(params[:user_id])
+    @posts = @user.posts
+  end
 
-  def show; end
+  def show
+    @user = User.find(params[:user_id])
+    @post = @user.posts.find(params[:id])
+    @comments = @post.comments
+    @likes = @post.likes
+  end
+
+  def new
+    @current_user = current_user
+    @new_post = Post.new
+  end
+
+  def create
```

Comment:
- [ ] After resolving the routes issues raised earlier, you may not be able to create a post, I got the error message screenshot below 👇; the reason for this is because `title` is implemented as **Title** in `schema.rb` file. 👍 

<img width="640" alt="Untitled" src="https://github.com/AlaaAlsalem/blogaapp/assets/103386183/26a54a11-0953-48a2-8d73-b20e594ae4bf">